config/environments/production.rb

```diff
@@ -28,7 +28,7 @@
   config.require_master_key = !IS_DOCKER_BUILD
 
   # Disable serving static files from `public/`, relying on NGINX/Apache to do so instead.
-  config.public_file_server.enabled = false
```

Comment:
We changed this to `false` here https://github.com/davidrunger/david_runger/pull/4884/files#diff-da60b4e96eff2b132991226d308949e23f4ef3aad45ad59edd09cbc32cc6251eR31 but probably shouldn't have.